### ISSUE TITLE
./build_setup needed with 2.0.12 tar

### ISSUE THREAD

**Issue by vadapalliravikumar** [acassen/keepalived#1116]
The 2.0.12 release does not come with a pre-done ./build_setup. Whereas the previous releases like 2.0.10 & 2.0.8 releases come with a pre-done ./build_setup. Is this intentional ?

**Comment by acassen** [comment]
./configure is here. but seems build_setup is here too. Not a big deal just run ./configure. Maybe a commit added build_setup file but it is not needed, will have a look.

**Comment by pqarmitage** [comment]
I added build_setup to the tarball in commit f42774f. The reason I did so was in case someone wanted to set up the automake/autoconf files based on their installed automake/autoconf rather than use the generated ones included in the tarball. 

It is still possible to execute `./configure; make` directly from the files extracted from the tarball.

**Comment by acassen** [comment]
yes build_setup is now shipped with tarball. just in case tarball user change something in configure.ac and want to rebuild the whole auto-stuff.

**Comment by acassen** [comment]
outch collision ;)

**Comment by vadapalliravikumar** [comment]
I see just ./configure.ac in 2.0.12, but both ./configure & ./configure.ac in 2.0.10

```
ravvadap@ravvadap-dev:/tmp$ tar -tvf keepalived-2.0.12.tar.gz  | grep configure
-rw-rw-r-- root/root     84497 2019-01-26 02:19 keepalived-2.0.12/configure.ac
ravvadap@ravvadap-dev:/tmp$ 
ravvadap@ravvadap-dev:/tmp$ tar -tvf keepalived-2.0.10.tar.gz  | grep configure
-rwxrwxr-x 1000/1000    377302 2018-11-12 11:40 keepalived-2.0.10/configure
-rw-rw-r-- 1000/1000     76707 2018-11-12 11:34 keepalived-2.0.10/configure.ac
ravvadap@ravvadap-dev:/tmp$

**Comment by pqarmitage** [comment]
Where are you getting keepalived-2.0.12.tar.gz from? The keepalived-2.0.12.tar.gz from http://www.keepalived.org/download.html has configure in the tarball. Also, if I create a tarball from the source tree using `make dist` there is configure in the tarball.
